pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -4,7 +4,7 @@ ARG MAMBA_VERSION=22.9.0-1
 ARG OPEN_MPI_VERSION=4.1.4
 
 # PyTorch Binaries
-ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.13.0_ec2/20221105-013447/41160d617a294a8963a868ec942a2569a2fbf3ea/torch-1.13.0%2Bcpu-cp39-cp39-linux_x86_64.whl
+ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.com/r1.13.0_ec2/20221205-202607/8cf843550346728d7e661cd0a26f74b603dc17a5/torch-1.13.0%2Bcpu-cp39-cp39-linux_x86_64.whl
```

Comment:
The binary name says `r1.13.0_ec2`. Does it just correspond to `ec2` images or `SM` as well?